### PR TITLE
AddIsCollapsed

### PR DESCRIPTION
**Pull request by Chentong-Xiang**

### CODE REVIEW

**Comment by Danielabom** [comment]
Reached out to the author to get info.

File: changelog/Microsoft.MicrosoftSearch.json [hunk 1]
```diff
@@ -670,6 +670,24 @@
...
       "SubArea": "Query"
     },
+    {
+      "ChangeList": [
+        {
+          "Id": "76a83fe0-9c28-47eb-82ac-a9909d8af214",
+          "ApiChange": "Property",
+          "ChangedApiName": "isCollapsed",
+          "ChangeType": "Addition",
+          "Description": "Added the `isCollapsed` property to the [searchHit](https://learn.microsoft.com/en-us/graph/api/resources/searchHit?view=graph-rest-beta) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.MicrosoftSearch.json:680]:
```suggestion
          "Description": "Added the **isCollapsed** property to the [searchHit](https://learn.microsoft.com/en-us/graph/api/resources/searchHit?view=graph-rest-beta) resource.",
```

**Review by Danielabom — APPROVED** [review]
Thank you for working on this PR @Chentong-Xiang. PR approved.

File: changelog/Microsoft.MicrosoftSearch.json [hunk 2]
```diff
@@ -670,6 +670,24 @@
...
       "SubArea": "Query"
     },
+    {
+      "ChangeList": [
+        {
+          "Id": "76a83fe0-9c28-47eb-82ac-a9909d8af214",
+          "ApiChange": "Property",
+          "ChangedApiName": "isCollapsed",
+          "ChangeType": "Addition",
+          "Description": "Added the **isCollapsed** property to the [searchHit](https://learn.microsoft.com/en-us/graph/api/resources/searchHit?view=graph-rest-beta) resource.",
```
Comment (Lauragra) [inline changelog/Microsoft.MicrosoftSearch.json:680]: Please update the searchHit resource topic to include the new property, and if applicable, any method topic examples if this property is returned in responses.

File: concepts/search-concept-collapse.md [hunk 3]
```diff
@@ -162,8 +162,54 @@ In below table, the ranking is still maintained, but we collapse first on Filena
 |Notebook|James|Science|6|
 |Notebook|Andy|Culture|7|
 
+## Expected Response
```
Comment (Lauragra) [inline concepts/search-concept-collapse.md:165]: We use sentence case for headings; please revise as "Expected response".

File: concepts/search-concept-collapse.md [hunk 4]
```diff
@@ -162,8 +162,54 @@ In below table, the ranking is still maintained, but we collapse first on Filena
...
+When using collapseProperties, the response will contain an "isCollapsed" Boolean field for each result that represents the collapse state of that result.
```
Comment (Lauragra) [inline concepts/search-concept-collapse.md:166]: Please revise this sentence as follows:

When you use **collapseProperties**, the response contains an **isCollapsed** Boolean property for each result. This property indicates the collapse state of the result.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**